importer/NlGemNl.py

```diff
@@ -0,0 +1,157 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class NlGemNl(Monument):
+
+    def set_adm_location(self):
+        """
+        Set administrative location.
+
+        Use the Municipality ID to resolve to wd item.
+        """
+        municipal_match = [
+            x["item"] for x in self.data_files["municipalities"]
+            if x["value"] == self.gemcode]
+        if len(municipal_match) == 1:
+            self.add_statement("located_adm", municipal_match[0])
+        else:
+            self.add_to_report("gemcode", self.gemcode, "located_adm")
+
+    def set_monuments_all_id(self):
+        """Map which column in table to ID in monuments_all."""
+        self.monuments_all_id = "{}/{}".format(self.gemcode, self.objnr)
+
+    def set_architect(self):
+        """
+        Set architect.
+
+        Only if it's wikilinked. There's a lot of free text
+        ones which cannot be matched to anything as they don't
+        have nlwp articles, only have initials not full names etc.
+        """
+        if self.has_non_empty_attribute("architect"):
+            if utils.count_wikilinks(self.architect) > 0:
```

Comment:
`if utils.count_wikilinks(self.architect)`: should work

Reply:
Done.